cloud/logger-cloud/test/BunyanLogger.ts

```diff
@@ -0,0 +1,29 @@
+/// <reference types="mocha" />
+'use strict';
+import * as bunyan from 'bunyan';
+import * as proxyquire from 'proxyquire';
+import { expect } from 'chai';
+
+// const BunyanLogger = proxyquire.noCallThru().load('../src/BunyanLogger', {
+//
+// });
+
+import {BunyanLogger} from '../src/BunyanLogger';
+const loggerObject:object = {test:"test"};
+const emptyObject:object = {};
+
+const log = new BunyanLogger({name:"test"});
+
+
+describe('Expected results', () => {
+ it('should return the expected responses', () => {
+   log.debug('debug logger message', loggerObject);
+   log.error('error logger message', loggerObject);
+   log.info('info logger message', loggerObject);
+   log.warn('warn logger message', loggerObject);
```

Comment:
In order for us to be able to test anything we probably need to have the interface receive an arbitrary stream (defaulting to stdout in implementations) that we could then mock and test, but that could be overkill for now